pages/packages/[package]/versions/[version]/topics/[topic].tsx

```diff
@@ -139,10 +139,28 @@ export default function TopicPage({ topicData }: Props) {
             <section>
               <div className="relative">
                 <h2>Examples</h2>
+                {navigator.maxTouchPoints < 1 && (
```

Comment:
There's no method that's 100%. I don't like screen width because if you have a resized window or resize it later, you won't see it. This was a recommended method from [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent)